### PR TITLE
Pam branch

### PR DESCRIPTION
**Pull request by pcrainer38**
added html for home page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,8 +4,50 @@
...
+              </figure>
+        </div>
+      </section>
+      <section class="search-options">
+        <div class="field has-addons columns is-centered mb-2">
+            <div class="control" id="search-city">
+            <input class="input" type="text" placeholder="Search by City">
+            </div>
+            <div class="control">
+            <a class="button is-info">
```
Comment (Bfinch3) [inline index.html:34]: Can you put this a into the div above, the same div as the input

File: index.html [hunk 2]
```diff
@@ -4,8 +4,50 @@
...
+            </a>
+            </div>
+        </div>
+      
+        <div class="field has-addons columns is-centered mt-2">
+            <div class="control" id="search-artist">
+            <input class="input" type="text" placeholder="Search by Artist">
+            </div>
+            <div class="control">
+            <a class="button is-info">
```
Comment (Bfinch3) [inline index.html:45]: Can you put this a into the div above, the same div as the input

**Review by Bfinch3 — CHANGES_REQUESTED** [review]
See comments for changes

File: index.html [hunk 3]
```diff
@@ -4,8 +4,50 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Anton&family=Rubik&display=swap" rel="stylesheet">
```
Comment (Bfinch3) [inline index.html:9]: Can you add the link to jQuery

**Comment by pcrainer38** [comment]
I can add jquery but if I move the div's it changes the structure of the search field and button.

**Pull request closed without merging**